common/tree/builder.go

```diff
@@ -22,15 +22,14 @@ const (
 // radix is hardcoded to 2
 func CraftSharedOutput(
 	receivers []Leaf,
-	feeSatsPerNode uint64,
 	sweepTapTreeRoot []byte,
 ) ([]byte, int64, error) {
-	root, err := createTxTree(receivers, feeSatsPerNode, sweepTapTreeRoot, vtxoTreeRadix)
+	root, err := createTxTree(receivers, sweepTapTreeRoot, vtxoTreeRadix)
 	if err != nil {
 		return nil, 0, err
 	}
 
-	amount := root.getAmount() + int64(feeSatsPerNode)
+	amount := root.getAmount() + ANCHOR_VALUE
```

Comment:
does it really make sense to keep this?

Reply:
it allows the anchor to be configurable in case we want to revert to anchor with value ?
it's just a constant, so I would keep it but as you prefer